dts/bindings/qspi/st,stm32-qspi.yaml

```diff
@@ -38,11 +38,21 @@ properties:
   dmas:
     description: |
       Optional DMA channel specifier. If DMA should be used, specifier should
-      hold a phandle reference to the dma controller, the channel number,
-      the slot number, channel configuration and finally features.
+      hold a phandle reference to the dma controller not the DMAMUX even if present,
+      the channel number, the slot number,channel configuration and finally features.
+      (depending on the type of DMA)
```

Comment:
Here there are 5 elements listed, but the example below contains only 4. I understand that some can be optional, but it may be useful to make this clearer